dotnet/docs/EXPERIMENTS.md

```diff
@@ -31,7 +31,7 @@ You can use the following diagnostic IDs to ignore warnings or errors for a part
 | SKEXP0001 | Embedding services | | | | | |
 | SKEXP0001 | Image services | | | | | |
 | SKEXP0001 | Memory connectors | | | | | |
-| SKEXP0001 | Kernel filters | | | | | |
+| SKEXP0001 | Kernel filters | [AutoFunctionInvocationContext Class](https://learn.microsoft.com/en-us/dotnet/api/microsoft.semantickernel.autofunctioninvocationcontext?view=semantic-kernel-dotnet) <br> [IAutoFunctionInvocationFilter Interface](https://learn.microsoft.com/en-us/dotnet/api/microsoft.semantickernel.iautofunctioninvocationfilter?view=semantic-kernel-dotnet) <br>[FunctionInvocationContext Class](https://learn.microsoft.com/en-us/dotnet/api/microsoft.semantickernel.functioninvocationcontext?view=semantic-kernel-dotnet) <br>[IFunctionInvocationFilter Interface](https://learn.microsoft.com/en-us/dotnet/api/microsoft.semantickernel.ifunctioninvocationfilter?view=semantic-kernel-dotnet) <br> [IPromptRenderFilter Interface](https://learn.microsoft.com/en-us/dotnet/api/microsoft.semantickernel.ipromptrenderfilter?view=semantic-kernel-dotnet) <br> [PromptRenderContext Class](https://learn.microsoft.com/en-us/dotnet/api/microsoft.semantickernel.promptrendercontext?view=semantic-kernel-dotnet) | | [Auto Function Invocation Filtering](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/Concepts/Filtering/AutoFunctionInvocationFiltering.cs) <br> [Function Invocation Filtering](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/Concepts/Filtering/FunctionInvocationFiltering.cs) <br> [Prompt Render Filtering](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/Concepts/Filtering/PromptRenderFiltering.cs) <br> [FrugalGPT with Filters](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/Concepts/Optimization/FrugalGPTWithFilters.cs) <br> [Plugin selection with Filters](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/Concepts/Optimization/PluginSelectionWithFilters.cs) <br> [PII Detection](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/Concepts/Filtering/PIIDetection.cs) <br> [Retry with Filters](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/Concepts/Filtering/RetryWithFilters.cs) <br> [Telemetry with Filters](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/Concepts/Filtering/TelemetryWithFilters.cs) <br> [Semantic Caching with Filters](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/Concepts/Caching/SemanticCachingWithFilters.cs) <br> [Azure AI Content Safety and Prompt Shields service](https://github.com/microsoft/semantic-kernel/tree/main/dotnet/samples/Demos/ContentSafety) <br> [Function Invocation Approval](https://github.com/microsoft/semantic-kernel/tree/main/dotnet/samples/Demos/FunctionInvocationApproval) <br> [Quality Check with Filters](https://github.com/microsoft/semantic-kernel/tree/main/dotnet/samples/Demos/QualityCheck) | | |
```

Comment:
Why do we keep this around when we have removed the experimental attribute for these classes?

Reply:
Agreed with @markwallace-microsoft and @matthewbolanos to remove this information from the document.